package-lock.json

```diff
@@ -44,6 +44,12 @@
             "resolved": "https://registry.npmjs.org/@nodelib/fs.stat/-/fs.stat-1.1.3.tgz",
             "integrity": "sha512-shAmDyaQC4H92APFoIaVDHCx5bStIocgvbwQyxPRrbUY20V1EYTbSDchWbuwlMG3V17cprZhA6+78JfB+3DTPw=="
         },
+        "@openapitools/openapi-generator-cli": {
+            "version": "0.0.9-3.3.4",
```

Comment:
Better to put 0.0.9-4.0.2

See https://github.com/jhipster/generator-jhipster/pull/9925

Reply:
updated to 0.0.14-4.0.2